backend/core/src/shared/email.service.ts

```diff
@@ -34,6 +37,43 @@ export class EmailService {
     await this.send(user.email, "Welcome to Bloom", this.template("register-email")({ user: user }))
   }
 
+  public async confirmation(user: User, listing: Listing, application: Application) {
+    let whatToExpectText
+    const listingUrl = `${process.env.APP_URL}/listing/${listing.id}`
```

Comment:
We might not want to address this now, but I know that in the past we've talked about having a single backend instance that serves multiple front ends. This would prevent us from using a single env variable to hold the front end `app_url` like this. We could perhaps have the client calling this API path provide a URL instead? (or inspect the `Request` object to pull it out?)

Reply:
Yeah, this will actually be the case as soon as we expand the email functionality from Alameda to SMC, since both sites share the same backend. I'm fine if we want to do a separate PR for that, so @software-project feel free to either address now or file an issue for later as you prefer.

In terms of implementation, I think I prefer having the front-end set it explicitly, since I've seen referrer URLs get messed up a bunch of times.

For the longer term, I think what we actually want to do is add a `site` field to the user and/or listing, and then use that, so we're set up for future notification emails that might not be triggered by an explicit user action. There seems like there could be a bunch of subtleties to get that right in the farther future where we could have listings that could appear on multiple sites (e.g. a city site, county site, and regional site, all from the same backend) and we (maybe?) *also* want user accounts to transcend those sites? I'd be up for any solution that solves this most of the way in the short term, and then we can build more when we need it.